dbfit-java/postgres/src/main/java/dbfit/environment/PostgresEnvironment.java

```diff
@@ -44,15 +45,14 @@ protected String parseCommandText(String commandText) {
 
     public Map<String, DbParameterAccessor> getAllColumns(String tableOrViewName)
             throws SQLException {
-        String[] qualifiers = NameNormaliser.normaliseName(tableOrViewName)
-                .split("\\.");
+        String[] qualifiers = tableOrViewName.split("\\.");
         String qry = " select column_name, data_type, character_maximum_length "
                 + "	as direction from information_schema.columns where ";
 
         if (qualifiers.length == 2) {
-            qry += " lower(table_schema)=? and lower(table_name)=? ";
+            qry += " table_schema=? and table_name=? ";
```

Comment:
> Just to put in correct way if you create table or columns without " postgres automatically convert it to lower case adn then it will by stored in information schema. I added that test into test suite.

Have you pushed that test, I don't see it yet?